### PR TITLE
Prepare Release v0.38.0

### PR DESCRIPTION
**Pull request by srenatus**
🎊

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,96 @@
...
+      ...
+    }
+
+The available keys are:
+
+- title
+- description
+- authors
+- organizations
+- related resources
```
Comment (anderseknert) [inline CHANGELOG.md:33]: related_resources ?

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,7 +3,96 @@
...
+- Planner: Fix bug related to undefined results in dynamic lookups
+
+### Documentation and Website
+
+- Policy Reference: update EBNF to include "every" and "some x in ..." ([#4216](https://github.com/open-policy-agent/opa/issues/4216))
+- REST API: Update docs on 400 response
+- README: Include Google Analytic Instructions
+- Envoy primer: use variables instead of objects
+- Istio tutorial: expose application to outside traffic
+- website: New "Community" Webpage (authored by @msorens)
```
Comment (anderseknert) [inline CHANGELOG.md:85]: "Website:" to follow the convention from above.

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,7 +3,96 @@
...
+### WebAssembly
+
+- wasm SDK: OPA now uses Wasmtime 0.34.0 to evaluate its Wasm modules.
```
Comment (anderseknert) [inline CHANGELOG.md:88]: Same.

**Review by anderseknert — COMMENTED** [review]
Yay! Only some nits :)

File: CHANGELOG.md (hunk 1 above)
Comment (srenatus) [reply CHANGELOG.md:33]: yeah well that's the key... but it's not... the meaning...? 🤔

Comment (srenatus) [reply CHANGELOG.md:33]: 🤦 but the line above says "keys". Will fix.

**Comment by srenatus** [comment]
ℹ️ The links will work once the tag is pushed.

File: CHANGELOG.md (hunk 1 above)
Comment (johanfylling) [reply CHANGELOG.md]: We explicitly say that we're listing keys up top, though 😉. The docs describe each key in more detail.

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,7 +3,97 @@
...
+
+### Tooling, SDK, and Runtime
+
+- Compile API: add `disableInlining` option ([#4357](https://github.com/open-policy-agent/opa/issues/4357)) reported and fixed by @srlk
+- Status API: add `http_code` to response ([#4259](https://github.com/open-policy-agent/opa/issues/4259)) reported and fixed by @jkbschmid
+- Status plugin: publish experimental bundle-related metrics via prometheus endpoint (authored by @rafaelreinert) -- See [Status Metrics](https://www.openpolicyagent.org/docs/v0.38.0/monitoring/#status-metrics) for details.
+- SDK: don't panic without config ([#4303](https://github.com/open-policy-agent/opa/issues/4303)) authored by @damienjburks
+- Storage: Support index for array appends (for JSON Patch compatibility)
+- `opa deps`: Fix pretty printed output to show virtual documents ([#4342](https://github.com/open-policy-agent/opa/issues/4342))
+### Rego and Topdown
```
Comment (johanfylling) [inline CHANGELOG.md:72]: Will the missing space above show in the render?

**Review by johanfylling — COMMENTED** [review]
LGTM!

File: CHANGELOG.md (hunk 4 above)
Comment (srenatus) [reply CHANGELOG.md:72]: I'll fix that in the dev-prep PR. thanks!

**Pull request merged**